project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -156,10 +160,40 @@ public static Document getDocumentByHash(String hash) {
       return null;
     }
 
-    ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
+    Document doc = getDocumentByHash(hash);
+    ArrayList<Long> userIDs = doc.getUserIDs();
+    //ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
     return userIDs;
   }
 
+  public static ArrayList<Long> getDocumentEditors(String hash) {
```

Comment:
Should these permission changes be split into another branch/PR? Not sure what this has to do with naming refactoring

Reply:
this was forked off the permissions branch; they should be de-linked now